resource_obmcs_identity_policy_test.go

```diff
@@ -67,6 +70,25 @@ func (s *ResourceIdentityPolicyTestSuite) TestAccResourceIdentityPolicy_basic()
 				Config: s.Config + `
 				resource "oci_identity_policy" "p" {
 					compartment_id = "${oci_identity_compartment.t.id}"
+					name = "-tf-policy"
+					description = "automated test policy"
+					statements = ["Allow group ${oci_identity_group.t.name} to read instances in    compartment ${oci_identity_compartment.t.name}"]
+				}`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(s.ResourceName, "id"),
+					resource.TestCheckResourceAttrSet(s.ResourceName, "compartment_id"),
+					resource.TestCheckResourceAttrSet(s.ResourceName, "time_created"),
+					resource.TestCheckResourceAttr(s.ResourceName, "name", "-tf-policy"),
+					resource.TestCheckResourceAttr(s.ResourceName, "policyHash", "5f251343112c766d4ef6014f0c97c68c"),
+					resource.TestCheckResourceAttr(s.ResourceName, "description", "automated test policy"),
+					resource.TestCheckResourceAttr(s.ResourceName, "statements.#", "1"),
```

Comment:
I don't think we should duplicate assertions from previous steps for values that are not being changed, unless there is a reasonable suspicion that some change might have side effects.

Reply:
removed, kept only assertions that are validates that field values has not changed